package.json

```diff
@@ -14,15 +14,15 @@
     "babel-plugin-transform-object-rest-spread": "6.26.0",
     "babel-preset-env": "1.7.0",
     "babel-preset-react": "6.24.1",
-    "eslint": "5.3.0",
+    "eslint": "^5.9.0",
```

Comment:
was this package.json accidentally added to this PR?

Reply:
Yeah, sort of. This seems like it should be updated? But likely in another PR? Generally confused about it tbh. I'll remove the package.json and package-lock.json from this PR and they can be addressed separately.